include/oneapi/dpl/pstl/utils_ranges.h

```diff
@@ -307,6 +309,43 @@ struct drop_view_simple
     }
 };
 
+template <typename _R, typename _Size>
+struct replicate_start_view_simple
```

Comment:
I think a semantic should be explained in the comment  before the struct, because it is not obvious.